design-proposals/vm-memory-dump.md

```diff
@@ -0,0 +1,147 @@
+# Overview
+Proposal for taking VM memory dump for analysis purposes.
+
+## Motivation
+One of the methods of troubleshooting guest virtual machines is getting a memory dump and analyzing it. As this is a standard capability in most VM management systems, it is also required for kubevirt’s VMs.
+
+## Goals
+Have a mechanism to get a VM memory dump.
```

Comment:
A memory dump including all disks attached to the VM is needed for some forensic tools IIUIC.

Reply:
as far as I understand adding the disks to the dump is not the goal, the tools I think we are focused on is volatility3 which doesnt need the disks it uses the memory dump as is